druid/src/command.rs

```diff
@@ -242,6 +242,13 @@ pub mod sys {
 
     /// Redo.
     pub const REDO: Selector = Selector::new("druid-builtin.menu-redo");
+
+    /// Notify descendens that focus node has changed focus
+    pub const FOCUS_NODE_FOCUS_CHANGED: Selector<bool> =
```

Comment:
What does the `bool` indicate?

Reply:
Nothing, I just don't know how to represent an empty type.

Reply:
No, my mistake, this `bool` indicates whether `focus_node` is focused or not.

Reply:
I'll fix comment

Reply:
And for the record if you don't need a type you would use the 'unit type', `()`.